### PR TITLE
[AF-1519] Added a second version of takari

### PR DESCRIPTION
**Pull request by desmax74**
@domhanak @jhrcek @porcelli @mareknovotny

### CODE REVIEW

**Comment by desmax74** [comment]
Boostrap https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/802

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

File: kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java [hunk 1]
```diff
@@ -53,13 +57,17 @@
 
     private FileSystemTestingUtils fileSystemTestingUtils = new FileSystemTestingUtils();
     private IOService ioService;
-    private String mavenRepo;
+    private Make.String mavenRepo;
```
Comment (porcelli) [inline kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java]: what is this?

File: kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java [hunk 2]
```diff
@@ -33,12 +33,16 @@
...
 import org.kie.workbench.common.services.backend.compiler.CompilationRequest;
 import org.kie.workbench.common.services.backend.compiler.CompilationResponse;
+import org.kie.workbench.common.services.backend.compiler.TestUtilGit;
 import org.kie.workbench.common.services.backend.compiler.TestUtilMaven;
 import org.kie.workbench.common.services.backend.compiler.configuration.MavenCLIArgs;
 import org.kie.workbench.common.services.backend.compiler.impl.BaseMavenCompiler;
 import org.kie.workbench.common.services.backend.compiler.impl.DefaultCompilationRequest;
 import org.kie.workbench.common.services.backend.compiler.impl.WorkspaceCompilationInfo;
 import org.kie.workbench.common.services.backend.utils.TestUtil;
+import org.mvel2.util.Make;
```
Comment (porcelli) [inline kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java]: what is this?

File: kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java (hunk 1 above)
Comment (desmax74) [reply kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java]: Idea automatic mess

File: kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java (hunk 2 above)
Comment (desmax74) [reply kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/impl/decorators/JGITCompilerBeforeDecoratorTest.java]: Idea automatic mess

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins execute full downstream build

**Comment by desmax74** [comment]
jenkins test

**Pull request merged**